Task 6/Task6.2/main.css

```diff
@@ -0,0 +1,33 @@
+body {
+    font-family: 'Roboto Slab', serif;
+    font-size: 12pt;
+}
+
+.registration {
+    background-color: cornflowerblue;
+    color: cornsilk;
+    width: 400px;
+    box-sizing: content-box;
+}
+
+.registration  .button-registration {
+    background-color: gold;
+    width: 100px;
+}
```

Comment:
* Don't set a limited width for this button - if localized, button text would go beyond it. Instead set it back to `auto` (or `initial`, which will reset it to default, which is `auto`).
* Remove default button borders